aries-site/src/components/home/Highlights.js

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment, useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  Paragraph,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+import Link from 'next/link';
+
+import { ContentPreviewCard } from '../cards';
+import { internalLink } from '../content';
+import { nameToPath, useDarkMode } from '../../utils';
+import { highlights } from '../../data';
+
+const HighlightsLayout = () => {
+  const size = useContext(ResponsiveContext);
+  const darkMode = useDarkMode();
+
+  return (
+    <Grid
+      pad={{ horizontal: size !== 'small' ? 'medium' : 'large' }}
+      columns={size !== 'small' ? 'medium' : '100%'}
+      rows={[['auto', 'full']]}
+      gap="large"
+      fill
+      justify="center"
+    >
+      {highlights.map(({ name, summary, image }) => {
+        const href = nameToPath(name);
+        const isInternalLink = internalLink.test(href);
+        const Wrapper = isInternalLink ? Link : Fragment;
+        const wrapperProps = isInternalLink && {
+          href,
+          passHref: true,
+        };
+
+        return (
+          <Wrapper key={name} {...wrapperProps}>
+            <ContentPreviewCard
+              forwardedAs="a"
+              style={{ textDecoration: 'none' }}
+              pad="medium"
+            >
+              <Box direction="row" gap="medium">
+                <Box width="small" round="xsmall">
+                  {image && (
+                    <Image
+                      src={
+                        darkMode.value
+                          ? image.src.dark || image.src
+                          : image.src.light || image.src
+                      }
+                      alt={image.alt}
+                      fit={image.fit || 'contain'}
+                    />
+                  )}
+                </Box>
+                <Box fill>
+                  <Text
+                    weight="bold"
+                    size="large"
+                    margin={{ top: 'small' }}
```

Comment:
Instead of `margin top` here, can we apply an "align=center" (or is it justify?) to the Box on line 62

Reply:
I think if you do that you can run into problems where cards with a different number of lines of text will look odd across a row because the titles aren't aligned. The idea here was to make sure the title was a consistent amount from the top. It could also be done with a pad={{ top: 'small' }} on 62 I think. Thoughts?

Reply:
Got it, thanks for the explanation. That makes sense, and I don't think align center would be right. I think your implementation is fine.